src/main/java/com/cascadebot/cascadebot/events/GeneralEvents.java

```diff
@@ -20,4 +25,27 @@ public void onReady(ReadyEvent event) {
         }
     }
 
+    @Override
+    public void onGuildJoin(GuildJoinEvent event) {
+        if (!StringUtils.isBlank(Config.INS.getGuildWelcomeMessage())) {
+            Guild guild = event.getGuild();
+            guild.getOwner().getUser().openPrivateChannel().queue(privateChannel -> {
+                privateChannel.sendMessage(Config.INS.getGuildWelcomeMessage()).queue();
+            }, error -> {
+                if (guild.getDefaultChannel() != null) {
```

Comment:
Aren't default channels not a thing any more? also we should send in a channel at all if the owners dms are closed imo.

Reply:
In JDA a default channel is defined as:
```
This channel is the first channel in the guild (ordered by position) that the getPublicRole() has the Permission.MESSAGE_READ in.
```
Good point about whether we should or not 🤔